src/libraries/System.Text.Json/tests/Serialization/ConstructorTests/ConstructorTests.Stream.cs

```diff
@@ -129,8 +139,12 @@ async Task RunTest<T>()
                 }
             }
 
-            await RunTest<Class_With_Ctor_With_65_Params>();
-            await RunTest<Struct_With_Ctor_With_65_Params>();
+            Task[] tasks = new Task[2];
+
+            tasks[0] = Task.Run(async () => await RunTestAsync<Class_With_Ctor_With_65_Params>());
+            tasks[1] = Task.Run(async () => await RunTestAsync<Struct_With_Ctor_With_65_Params>());
+
+            await Task.WhenAll(tasks);
```

Comment:
You made these run in parallel just for fun? :)

Reply:
These were actually already running in parallel before this PR.